tests/test_smoke.py

```diff
@@ -647,12 +678,15 @@ def test_autodown():
     test = Test(
         'autodown',
         [
-            f'sky launch -y -d -c {name} --num-nodes 2 --cloud gcp examples/minimal.yaml',
+            f'sky launch -y -d -c {name} --num-nodes 2 --cloud aws examples/minimal.yaml',
             f'sky autostop -y {name} --down -i 1',
             # Ensure autostop is set.
             f'sky status | grep {name} | grep "1m (down)"',
-            'sleep 240',
+            # Ensure the cluster is not terminated early.
+            'sleep 45',
+            f'sky status --refresh | grep {name} | grep UP',
             # Ensure the cluster is terminated.
+            'sleep 240',
```

Comment:
How about reducing this to `200`, as we have waited for 45 seconds in L686?

Reply:
Thanks, done and checked the test passed.